.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature Request
+about: Suggest an idea for this project	about
+title: ''
+labels: kind/feature-request
+assignees: ''
+
+---
+
+**Problem**
+A clear and concise description of what the problem or the user story this feature will solve. Ex. I'm always frustrated when [...]
+
+ **User Persona**
+What user persona does this feature address? These could be multiple. If possible, define "I can" statements for the user persona in 'Problem' and/or 'Exit criteria' section.
+
+ **Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+ **Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+ **Estimate**
+Estimate interms of in 1-engineer-day, that will take for this work to complete and meet exit criteria.
```

Comment:
in terms

Reply:
> How does github let the user select these templates? I have not seen that before.
> 
> I think the whole thing is way too wordy. Cut about 1/2 I think. Be more concise.

When you click on new issue you will see these options.